CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [#116](https://github.com/slack-ruby/slack-ruby-client/pull/116): Use [slack-ruby/slack-api-ref](https://github.com/slack-ruby/slack-api-ref) as machine API reference - [@dblock](https://github.com/dblock).
 * [#116](https://github.com/slack-ruby/slack-ruby-client/pull/116): Added `users_setPhoto` and `users_deletePhoto` to Web API - [@dblock](https://github.com/dblock).
 * [#81](https://github.com/slack-ruby/slack-ruby-client/pull/81): Require faraday 0.9.0 or newer - [@leppert](https://github.com/leppert).
+* [#119](https://github.com/slack-ruby/slack-ruby-client/pull/119): Fix typo rest -> reset - [@naofumi-fujii](https://github.com/naofumi-fujii).
```

Comment:
No need to CHANGELOG a typo! Danger is being overzealous. You can remove this please and sorry for the noise.